stone/target/python_type_mapping.py

```diff
@@ -0,0 +1,91 @@
+from stone.api import ApiNamespace
+from stone.data_type import (
+    Alias,
+    DataType,
+    List,
+    Nullable,
+    Timestamp,
+    UserDefined,
+    is_alias,
+    is_boolean_type,
+    is_bytes_type,
+    is_float_type,
+    is_integer_type,
+    is_list_type,
+    is_nullable_type,
+    is_string_type,
+    is_timestamp_type,
+    is_user_defined_type,
+    is_void_type,
+)
+from stone.target.python_helpers import class_name_for_data_type
+from stone.typing_hacks import cast
+
+MYPY = False
```

Comment:
Gosh, I'm getting used to the niceness of doing everything new in Python 3, that I think I might just make Stone's generator-interface PY3 only (of course we still need to generate PY 2 code...).